Wrapping/Generators/Python/Tests/extras.py

```diff
@@ -344,6 +344,11 @@ def custom_callback(name, progress):
     cast = image.astype(np.float32)
     assert cast.dtype == np.float32
 
+    npImage = (np.random.rand(12,8,3)*256).astype(np.uint8)
+    image = itk.image_from_array(npImage, is_vector=True)
+    cast = image.astype(itk.RGBPixel[itk.US])
```

Comment:
`itk.RGBPixel[itk.US]` -> `itk.RGBPixel[itk.UC]` to use a default wrapped pixel type, which should address the failing test:

```
<_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'> ImageFileReader (0x7fa376f3efd0)
  RTTI typeinfo:   itk::ImageFileReader<itk::Image<unsigned char, 2u>, itk::DefaultConvertPixelTraits<unsigned char> >
  Reference Count: 1
  Modified Time: 76
  Debug: Off
  Object Name: 
  Observers: 
    none
  Inputs: 
    FileName: (0x7fa376f3f4a0)
    Primary: (0x0)
  Indexed Inputs: 
    0: Primary (0x0)
  No Required Input Names
  NumberOfRequiredInputs: 0
  Outputs: 
    Primary: (0x7fa376f3f2e0)
  Indexed Outputs: 
    0: Primary (0x7fa376f3f2e0)
  NumberOfRequiredOutputs: 1
  Number Of Work Units: 16
  ReleaseDataFlag: Off
  ReleaseDataBeforeUpdateFlag: Off
  AbortGenerateData: Off
  Progress: 0
  Multithreader: 
    RTTI typeinfo:   itk::PoolMultiThreader
    Reference Count: 1
    Modified Time: 60
    Debug: Off
    Object Name: 
    Observers: 
      none
    Number of Work Units: 16
    Number of Threads: 4
    Global Maximum Number Of Threads: 128
    Global Default Number Of Threads: 4
    Global Default Threader Type: itk::MultiThreaderBaseEnums::Threader::Pool
    SingleMethod: 0
    SingleData: 0x0
  DynamicMultiThreading: On
  ImageIO: (null)
  UserSpecifiedImageIO flag: 0
  m_UseStreaming: 1

Traceback (most recent call last):
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itkTemplate.py", line 349, in __getitem__
    return self.__template__[tuple(cleanParameters)]
KeyError: (<class 'itk.itkRGBPixelPython.itkRGBPixelUS'>, 2)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itkTemplate.py", line 353, in __getitem__
    return self.__template__[tuple(cleanParameters)]
KeyError: (<class 'itk.itkRGBPixelPython.itkRGBPixelUS'>, 2)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "extras.py", line 349, in <module>
    cast = image.astype(itk.RGBPixel[itk.US])
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itk/itkImagePython.py", line 3069, in astype
    OutputImageType = current_image_type[pixel_type, current_image_dimension]
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itkTemplate.py", line 355, in __getitem__
    raise TemplateTypeError(self, tuple(cleanParameters))
itkTemplate.TemplateTypeError: itk.Image is not wrapped for input type `itk.RGBPixel[itk.US], int`.

To limit the size of the package, only a limited number of
types are available in ITK Python. To print the supported
types, run the following command in your python environment:

    itk.Image.GetTypes()

Possible solutions:
* If you are an application user:
** Convert your input image into a supported format (see below).
** Contact developer to report the issue.
* If you are an application developer, force input images to be
loaded in a supported pixel type.

    e.g.: instance = itk.Image[itk.RGBPixel[itk.UC], int].New(my_input)

* (Advanced) If you are an application developer, build ITK Python yourself and
turned to `ON` the corresponding CMake option to wrap the pixel type or image
dimension you need. When configuring ITK with CMake, you can set
`ITK_WRAP_${type}` (replace ${type} with appropriate pixel type such as
`double`). If you need to support images with 4 or 5 dimensions, you can add
these dimensions to the list of dimensions in the CMake variable
`ITK_WRAP_IMAGE_DIMS`.

Supported input types:

itk.RGBPixel[itk.UC]
itk.RGBAPixel[itk.UC]
itk.Vector[itk.F,2]
itk.Vector[itk.F,3]
itk.Vector[itk.F,4]
itk.CovariantVector[itk.F,2]
itk.CovariantVector[itk.F,3]
itk.CovariantVector[itk.F,4]
itk.SS
itk.UC
itk.F
itk.complex[itk.F]
itk.Vector[itk.D,2]
itk.Vector[itk.D,3]
itk.Vector[itk.D,4]
itk.D
itk.UL
itk.ULL
itk.B
itk.FixedArray[itk.F,2]
itk.Offset[2]
itk.SymmetricSecondRankTensor[itk.D,2]
itk.RGBPixel[itk.UC]
itk.RGBAPixel[itk.UC]
itk.Vector[itk.F,2]
itk.Vector[itk.F,3]
itk.Vector[itk.F,4]
itk.CovariantVector[itk.F,2]
itk.CovariantVector[itk.F,3]
itk.CovariantVector[itk.F,4]
itk.SS
itk.UC
itk.F
itk.complex[itk.F]
itk.Vector[itk.D,2]
itk.Vector[itk.D,3]
itk.Vector[itk.D,4]
itk.D
itk.UL
itk.ULL
itk.B
itk.FixedArray[itk.F,3]
itk.Offset[3]
itk.SymmetricSecondRankTensor[itk.D,3]
itk.Vector[itk.F,1]
itk.Vector[itk.D,2]
itk.Vector[itk.D,3]
itk.Vector[itk.D,4]
itk.Vector[itk.F,1]
itk.CovariantVector[itk.D,2]
itk.CovariantVector[itk.D,3]
itk.CovariantVector[itk.D,4]
itk.CovariantVector[itk.D,2]
itk.CovariantVector[itk.D,3]
itk.CovariantVector[itk.D,4]
itk.NormalBandNode[itk.Image[itk.F,2]]
itk.NormalBandNode[itk.Image[itk.F,3]]
```

Reply:
The test is now casting `itk.Image[itk.Vector[itk.F,3],2]` to `itk.Image[itk.Vector[itk.D,3],2]` with `cast = image.astype(itk.Vector[itk.D,3])`.